app/controllers/profiles_controller.rb

```diff
@@ -29,6 +29,17 @@ def update
     end
   end
 
+  def delete
+    @only_owner_gems = current_user.only_owner_gems
+    @multi_owner_gems = current_user.rubygems_downloaded - @only_owner_gems
+  end
+
+  def destroy
+    Delayed::Job.enqueue DeleteUser.new(current_user), priority: PRIORITIES[:web_hook]
```

Comment:
`priority: PRIORITIES[:web_hook]`

What about to create new priority for this action (even with same value as :web_hook)?